pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -146,16 +151,16 @@ func (gc *GarbageCollector) Run(ctx context.Context, workers int) {
 	defer gc.attemptToOrphan.ShutDown()
 	defer gc.dependencyGraphBuilder.graphChanges.ShutDown()
 
-	klog.Infof("Starting garbage collector controller")
-	defer klog.Infof("Shutting down garbage collector controller")
+	klog.Infof("%s: Starting garbage collector controller", gc.clusterName)
```

Comment:
Not critical, but a slightly easier approach would be to make a commit that moves this controller to use contextual logging, which helps the upstream efforts and can land in k8s quickly, then simply add the cluster name to the logger we pass to the context that we use to `Run()` the controller

Reply:
Would like to proceed now. We can revert this later and adapt on top of contextual logging when my PR lands upstream.